### ISSUE TITLE
Workspace.getActiveEditor is deprecated.

### ISSUE THREAD

**Issue by envygeeks** [lee-dohm/auto-copyright#10]
Call ::getActiveTextEditor instead
```
Workspace.getActiveEditor (/opt/github/atom/share/atom/resources/app.asar/src/workspace.js:1022:12)
AutoCopyright.insert (/home/jordon/.atom/packages/auto-copyright/lib/auto-copyright.coffee:33:28)
atom-workspace.atom.commands.add.auto-copyright:insert (/home/jordon/.atom/packages/auto-copyright/lib/auto-copyright.coffee:27:9)
CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/opt/github/atom/share/atom/resources/app.asar/src/command-registry.js:238:29)
```

**Comment by lee-dohm** [comment]
Fixed in #12

**Comment by lee-dohm** [comment]
Thanks!
